plugins/resources/unixfilesystem/src/irods_is_in_host_list.cpp

```diff
@@ -0,0 +1,52 @@
+#include "irods/private/irods_is_in_host_list.hpp"
+
+#include "irods/filesystem/path.hpp"
+#include "irods/irods_error.hpp"
+#include "irods/irods_logger.hpp"
```

Comment:
This header can be removed since it's not used in the implementation.

Reply:
done